src/autoschedulers/anderson2021/LoopNest.cpp

```diff
@@ -0,0 +1,4001 @@
+                int64_t innermost_storage_extent = 1;
+                int v = p->vector_dim;
+                if (v >= 0 && v < node->dimensions) {
+                    innermost_storage_extent = root_bounds->region_computed(v).extent();
+                }
+                feat.innermost_bytes_at_root = node->bytes_per_point * innermost_storage_extent;
+            } else {
+                feat.innermost_bytes_at_root = 0;
+            }
+
+            feat.points_computed_minimum = 1;
+            for (int i = 0; i < (int)stage->loop.size(); i++) {
+                const auto &p = root_bounds->loops(stage->index, i);
+                feat.points_computed_minimum *= p.extent();
+            }
+
+            if (node->stages.size() == 1 && !node->is_output) {
+                int64_t points_computed_minimum_if_inlined = 0;
+                for (auto *e : node->outgoing_edges) {
+                    points_computed_minimum_if_inlined += features->get(e->consumer).points_computed_minimum * e->calls;
+                }
+                feat.points_computed_minimum = std::min(feat.points_computed_minimum, (double)points_computed_minimum_if_inlined);
+            }
+
+            // When memoizing, we need to recompute features for inlined Funcs
+            // so we reset them here
+            if (use_memoized_features && sites.get(stage).inlined) {
+                feat.inlined_calls = 0;
+                feat.num_scalars = 0;
+                feat.innermost_pure_loop_extent = 0;
+                feat.outer_parallelism = 0;
+                feat.num_warps_per_block = 0;
+                feat.num_threads_per_block = 0;
+                feat.points_computed_per_thread = 0;
+            }
+        }
+
+        if (use_memoized_features) {
+            for (const auto &c : children) {
+                uint64_t hash_of_producers = sites.get(c->stage).hash_of_producers_stored_at_root;
+
+                // When computing feat.points_computed_minimum above, the order
+                // of nodes considered is possibly different from the loop nest
+                // traversal order so 'features->get(e->consumer).points_computed_minimum'
+                // may not have been computed when it is accessed as a memoized
+                // feature. We memoize 'points_computed_minimum' here to ensure
+                // its value is always available
+                if (c->features.count(hash_of_producers) > 0) {
+                    c->memoize_points_computed_minimum(c->features[hash_of_producers], features);
+                }
+            }
+
+            recompute_inlined_features(sites, features);
+        }
+
+        return;
+    }
+
+    int64_t subparallelism = parallel_tasks * parallelism;
+
+    // Figure out the features at the compute_at level
+    internal_assert(!stage->node->is_input);
+    ScheduleFeatures &feat = features->get_or_create(stage);
+
+    if (innermost) {
+    } else {
```

Comment:
Why is this clause empty? Either remove it entirely or add a comment explaining why it is empty